common/jdbc/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'scala'
+    id 'org.springframework.boot'
+}
+
+apply from: project(":").file('gradle/java.gradle')
+
+dependencies {
+    implementation project(':common:rest-jdbc-spring-api')
+    implementation group: 'ai.h2o', name: 'mojo2-runtime-api'
+    implementation group: 'ai.h2o', name: 'mojo2-runtime-impl'
+    implementation group: 'ai.h2o', name: 'sparkling-water-core_2.11'
+    implementation group: 'ai.h2o', name: 'sparkling-water-scoring_2.11'
+    implementation group: 'org.scala-lang', name: 'scala-library'
+    implementation group: 'org.apache.spark', name: 'spark-core_2.11'
+    implementation group: 'org.apache.spark', name: 'spark-sql_2.11'
+    implementation group: 'org.apache.spark', name: 'spark-mllib_2.11'
+    implementation group: 'com.typesafe', name:'config'
+
+    zinc group: 'com.typesafe.zinc', name: 'zinc', version: '0.3.9'
+    zinc group: 'org.scala-lang', name: 'scala-library', version: '2.10.5'
+
+    testCompile group: 'org.scalatest', name: 'scalatest_2.11', version: '3.0.5'
+    testRuntimeOnly group:'org.scala-lang.modules', name: 'scala-xml_2.11', version: '1.1.1'
+    testImplementation group: 'com.google.truth.extensions', name: 'truth-java8-extension'
+    testImplementation group: 'org.mockito', name: 'mockito-core'
+    testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api'
+    testRuntime group: 'org.junit.jupiter', name: 'junit-jupiter-engine'
+}
+
+test {
+    useJUnitPlatform()
+}
+
+bootJar {
+    enabled=false
```

Comment:
Why do you need `bootJar` section?

Reply:
by default `bootJar` is set `enabled=true` and given that we need to add to the classpath for the scorer to work properly, this needed to be set to `false`